CoffeeChat/Controller/MatchViewController.swift

```diff
@@ -62,51 +111,50 @@ class MatchViewController: UIViewController {
         matchSummaryTableView.dataSource = self
         matchSummaryTableView.register(MatchSummaryTableViewCell.self, forCellReuseIdentifier: MatchSummaryTableViewCell.reuseIdentifier)
         view.addSubview(matchSummaryTableView)
-
-        reachOutButton.backgroundColor = .backgroundOrange
-        reachOutButton.setTitle("Reach out!", for: .normal)
-        reachOutButton.setTitleColor(.white, for: .normal)
-        reachOutButton.titleLabel?.font = ._20CircularStdBold
-        reachOutButton.layer.cornerRadius = reachOutButtonSize.height/2
-        reachOutButton.addTarget(self, action: #selector(reachOutPressed), for: .touchUpInside)
-        view.addSubview(reachOutButton)
-
-        setupConstraints()
-    }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        navigationController?.navigationBar.isHidden = true
     }
 
     private func setupConstraints() {
-        let padding: CGFloat = 35 // TODO: Not sure about dimensions.
-        let reachOutPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 70) // TODO: Not sure about dimensions.
-
-        matchDemographicsLabel.snp.makeConstraints { make in
-            make.leading.trailing.equalTo(matchNameLabel)
-            make.top.equalTo(matchNameLabel.snp.bottom).offset(8)
-        }
-
-        matchNameLabel.snp.makeConstraints { make in
-            make.leading.equalTo(matchProfileImageView.snp.trailing).offset(20)
-            make.trailing.equalToSuperview().inset(padding)
-            make.top.equalTo(matchProfileImageView).offset(6)
+        let padding: CGFloat = 35
+        let reachOutPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 70)
+        let logoutPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 30)
+        let meetupPadding = 24
+        let meetupWidth: CGFloat = 319
```

Comment:
Remove these explicit type annotations if you don't need them. Maybe the compiler can infer the type from the calls below? Maybe not.